migrations/1754332200.sh

```diff
@@ -1,2 +1,2 @@
 echo "Remove old Omarchy TUI app now that we have the Omarchy Menu"
-rm ~/.local/share/applications/omarchy.desktop
+rm ~/.local/share/applications/omarchy.desktop 2> /dev/null
```

Comment:
Would be better to check if the file exists before trying to `rm` - that way if it does exist and there is a legit error when removing it, the error wouldn't be silenced.